yt/fields/tensor_fields.py

```diff
@@ -0,0 +1,36 @@
+from functools import partial
+
+ax = "txyz"
```

Comment:
could this be moved closer to usage ?

Reply:
Sorry, could you elaborate? Not sure what you mean

Reply:
Ah, see what you mean. It's there because I anticipated filling this file with other stuff eventually. But now I have a different idea.